src/foam/u2/md/OverlayDropdown.js

```diff
@@ -126,7 +126,7 @@ foam.CLASS({
         this.left = 'auto';
         this.right = screenWidth - x + 10;
       } else {
-        this.left = x + 10;
+        this.left = x - 75;
```

Comment:
Should be this dependant on the size of the dropdown? Could test with the user dropdown on the bottom left of the sidebar